octavia_f5/restclient/bigip/bigip_restclient.py

```diff
@@ -71,9 +71,26 @@ def scheme(self):
 
     @property
     def is_active(self):
+        """
+        Get active device which is MASTER device in F5 devices pair.
```

Comment:
Are you referring to the `role` column of the `amphora` DB table? If so, this is an implementation detail. If you're referring to F5 terminology, I think you probably mean `active device`.

Reply:
Nope, I'm checking real-time status by request, I do not want to use timeless cache in the database provided by another process. The doc-strings added here means exactly what the function is doing: this function checking F5 Active/Backup status of device.

Reply:
I understand that. My question was about the word `MASTER`, which is not F5 terminology. The only place it's being used is in the amphora table. In this line it can be changed to `active`.

Reply:
Done